test/end-to-end/test_pkg_download.sh

```diff
@@ -7,21 +7,26 @@
 # for cleaner code and more fine grained tests. However, they are a
 # bit of a pain to program in bash. This is intended to provide
 # minimal testing pending our figuring out the best approach for
-# command line testing. 
+# command line testing.
 #
+# Assumptions:
+# 1. HAB_AUTH_TOKEN Environment variables is set and valid
```

Comment:
I don't even think you need a token at all, since all this is downloading stuff (though downloading _private_ packages is perhaps another worthwhile usecase to test in the future!)

Reply:
Agreed. Addressed here https://github.com/habitat-sh/habitat/pull/7104/commits/f6d3c4bb03a461c5282e7607e61682d1cbedbb1d